Aquality.Selenium/src/Aquality.Selenium/Elements/Key.cs

```diff
@@ -0,0 +1,263 @@
+﻿namespace Aquality.Selenium.Elements.Actions
+{
+    /// <summary>
+    /// Represents action keys which could be used to <see cref="Interfaces.IElement.SendKey(Key)"/>.
+    /// Used to enhance logging of SendKeys actions
+    /// Directly related to <see cref="OpenQA.Selenium.Keys"/>
+    /// </summary>
+    public enum Key
```

Comment:
Why do we need to define our own enum with keys? Can't we use the enum from Selenium? https://www.selenium.dev/selenium/docs/api/java/org/openqa/selenium/Keys.html

Reply:
As stated in the summary, we need this enum to enhance logging. There is no such enum in dotnet, it's a static class, and we cannot log it's values.